rcsb/utils/dictionary/DictMethodCommonUtils.py

```diff
@@ -1771,6 +1785,8 @@ def getDatabaseNameMap(self):
             "UNP": "UniProt",
             "GB": "GenBank",
             "PDB": "PDB",
+            # "PDB-DEV": "PDB-Dev",
+            # "PDB-IHM": "PDB-IHM",
```

Comment:
Will these be needed later on, or can they be removed?

Reply:
These can be deleted once all IHM entries are remediated.

Reply:
Can you add that note to these lines then?
```suggestion
            # Can be deleted once all IHM entries are remediated 
            # "PDB-DEV": "PDB-Dev",
            # "PDB-IHM": "PDB-IHM",
```